.github/ISSUE_TEMPLATE/feature.yml

```diff
@@ -10,6 +10,6 @@ body:
     validations:
       required: true
   - type: textarea
-    id: description
+    id: approach-and-alternatives
```

Comment:
I'm open to however we want the template to look or if we'd like to re-word here. I just wanted to have it be valid first.